openreview/tools.py

```diff
@@ -51,6 +53,99 @@ def create_profile(client, email, first, last, middle = None):
         raise openreview.OpenReviewException('There is already a profile with this email: {0}'.format(email))
 
 
+def get_or_create_profile(client, email, first, last, allow_duplicates=False, verbose=False):
```

Comment:
I think create profile is doing this, why we need another funciton?

Reply:
ah yes, you're right. I'm not sure why I missed that

Reply:
don't worry it is not urgent to fix this.